test/e2e/Dockerfile

```diff
@@ -34,7 +34,6 @@ RUN curl -fLO https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-cs
 
```

Comment:
If you update this to `<sha>/manifest_staging/deploy/` and for

SECRET_STORE_VERSION to be ad965e92d14a87ad88f7f61c764afaede1ff6107 or above